app/controllers/scc_accounts_controller.rb

```diff
@@ -126,4 +132,11 @@ def action_permission
       super
     end
   end
+
+  # Function filters valid products and removes all products without valid repositories
+  # The @scc_accounts and their products are set in the index function.
+  # The order call is necessary to apply the ordering to repository that have already been loaded from the database.
+  def scc_filtered_products
+    @scc_filtered_products = @scc_account.scc_products.where(product_type: 'base').order(:friendly_name)
+  end
```

Comment:
This helper function is accessed from the view to get the filtered products (which are set to @scc_account_products in the index function above). Can this be improved?

Reply:
I guess the alternative is to set it as a `before_action` for the specific view-action.

Reply:
I do not think you have to set `@scc_filtered_products` here btw..
IMHO the helper function should automatically return the return-value of the expression executed last.